### PR TITLE
assignment unique id annotation

### PR DESCRIPTION
**Pull request by burnout87**
an adjustment to the way the hash of the Run annotation was assigned

### CODE REVIEW

**Comment by burnout87** [comment]
when producing two different annotations from two different calls (eg `query_region` and `get_images`), but from the same module, would end up not generating two distinct annotations, rather the latter overwriting the former

this because the identifier for the annotation used was `self.__hash__()`, which was always the same when staying within the same module

I noticed this behavior when trying to generate a graph using the `Legacysurvey_tap.ipynb` notebook

still, probably also this one is not the best, but I could generate two distinct annotations

File: aqsconverters/aq.py [hunk 1]
```diff
@@ -178,7 +179,7 @@ def produce_annotation(self, aq_query_type, *args, **kwargs):
         run.aq_kwargs = kwargs
 
         log_renku_aqs(
-            RunSchema().dumps(run), str(self.__hash__()), 
+            RunSchema().dumps(run), str(run_id),
```
Comment (volodymyrss) [inline aqsconverters/aq.py:183]: Ok, makes sense. 

By the way, I am now a bit confused that there is a new run for each `produce_annotation` i.e. for each separate `aqs_query_object` et al. But that's a different point. Maybe a TODO?

Presumably, run should be once for `renku run`.

**Review by volodymyrss — APPROVED** [review]
Just one circumstantial point, why many runs per `renku run`?.. We'll need to take care of this at some point.

And at that time, it will not be possible to use true `run_id` as file name, since it will be reused per `renku run`.

File: aqsconverters/aq.py (hunk 1 above)
Comment (burnout87) [reply aqsconverters/aq.py:183]: each execution of `produce_annotation` instantiate a new `Run` object, is that what you meant ?

Comment (volodymyrss) [reply aqsconverters/aq.py:183]: as of now - yes. I am not sure this is what renku `Run` is meant to represent. If anything, this is more like `AstroQuery` - a request. There can be many of those per `Run`. They should be linked to `Run`. Maybe later.

Comment (burnout87) [reply aqsconverters/aq.py:183]: I will add a TODO to keep it in mind

**Comment by volodymyrss** [comment]
I am slightly perplexed why we did not merge #5 yet! Then all PRs would go on master, not on `simple-autolog`.

**Pull request merged**